src/she/native_dialogs.h

```diff
@@ -25,6 +25,7 @@ namespace she {
     virtual std::string fileName() = 0;
     virtual void setFileName(const std::string& filename) = 0;
     virtual bool show(Display* parent) = 0;
+    static std::string& g_lastUsedDir() { static std::string lastUsedDir{""}; return lastUsedDir; }
```

Comment:
Also a std::string doesn't need to be initialized with an empty string (as it's already initialized empty).